site/src/theme/typography.ts

```diff
@@ -0,0 +1,90 @@
+import { ThemeOptions } from "@mui/material";
+import { customColors } from "./palette";
+import { defaultTheme } from "./util";
+
+const FALLBACK_FONTS = [`"Helvetica"`, `"Arial"`, "sans-serif"];
```

Comment:
This is just an opinion but, I’d just use 

```suggestion
const fallbackFonts = [`"Helvetica"`, `"Arial"`, "sans-serif"];
```

here. If there is no difference between this const and the next one, why use different notations? This is something a newjoiner in JS would 100% ask and I don’t think we’d be able to provide a good answer. It’s better to save time on this and talk about something more interesting instead 😁

Personally, I’m a fan of banning `UPPER_CASE` variable names in codebases via [`@typescript-eslint/naming-convention`](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/naming-convention.md). But this is an optional suggestion because we haven’t agreed on this stuff 🙂

Reply:
Don't have strong feelings on this, have changed it to `fallbackFonts` here https://github.com/blockprotocol/blockprotocol/pull/203/commits/d8976f9c232272a858d6ab87f6b2b5520a855853